src/index.js

```diff
@@ -7,13 +7,16 @@ import { AddItems } from './pages/Add-items/Add-items';
 import { List } from './pages/List/List';
 import './index.css';
 import * as serviceWorker from './serviceWorker';
+import { ProductDetails } from './components/ProductDetails';
 
 const rootElement = document.getElementById('root');
 render(
   <BrowserRouter>
     <Routes>
       <Route path="/" element={<App />} />
-      <Route path="list" element={<List />} />
+      <Route path="list" element={<List />}>
```

Comment:
`'/list'`?

Reply:
I've been trying to learn a bit more about React router and following [this tutorial](https://reactrouterdotcom.fly.dev/docs/en/v6/getting-started/tutorial#add-some-routes) I came across this example:

`<BrowserRouter>
    <Routes>
      <Route path="/" element={<App />} />
      <Route path="expenses" element={<Expenses />} />
      <Route path="invoices" element={<Invoices />} />
    </Routes>
  </BrowserRouter>`

I wondered the same (why they didn't use an absolute path just to be safe) but forgot to do it in the end. Do you think it's a best practice?

Reply:
I haven't dove deep into react router v6, thanks!